lib/ellis.rb

```diff
@@ -54,9 +54,14 @@ def self.destroy_leaked_numbers(domain)
                         email: EMAIL,
                         password: "Please enter your details")
     cookie = r.cookies
-    r = RestClient::Request.execute(method: :get,
-                                    url: ellis_url(domain, "accounts/#{EMAIL}/numbers"),
-                                    cookies: cookie)
+    begin
+      r = RestClient::Request.execute(method: :get,
+                                      url: ellis_url(domain, "accounts/#{EMAIL}/numbers"),
+                                      cookies: cookie)
+    rescue RestClient::Exception => e
+      puts "Listing existing numbers failed with HTTP code #{e.http_code}, body #{e.http_body}"
```

Comment:
+1 for stopping this crashing.

I'm not convinced that the body is useful information though (it's hard to parse and you probably need to go to the Ellis logs anyway to get the context)